File: DataSummitSolution/DataSummitUI/src/app/efModels/azureCompanyResourceUrl.ts
import { Company } from "../companies/models/company";

export class AzureCompanyResourceUrl {

	AzureCompanyResourceUrlId: number;
	Name: string;
	Url: string;
	Key: string;
	ResourceType: string;
	CompanyId: number;
	Company: Company;

	constructor(azureCompanyResourceUrlId?: number, name?: string, url?: string,
		key?: string, resourceType?: string, companyId?: number,
		company?: Company)
	{
		this.AzureCompanyResourceUrlId = azureCompanyResourceUrlId || 0;
		this.Name = name || "";
		this.Url = url || "";
		this.Key = key || "";
		this.ResourceType = resourceType || "";
		this.CompanyId = companyId || 0;
		this.Company = company;
	}
}

File: DataSummitSolution/DataSummitModels/DB/AzureCompanyResourceUrl.cs
using System;
using System.Collections.Generic;

#nullable disable

namespace DataSummitModels.DB
{
    public partial class AzureCompanyResourceUrl
    {
        public long AzureCompanyResourceUrlId { get; set; }
        public string Name { get; set; }
        public string Url { get; set; }
        public string Key { get; set; }
        public string ResourceType { get; set; }
        public int CompanyId { get; set; }

        public virtual Company Company { get; set; }
    }
}
